test/framework/snow.go

```diff
@@ -22,6 +22,10 @@ const (
 	snowPodCidr          = "T_SNOW_POD_CIDR"
 	snowCredentialsFile  = "EKSA_AWS_CREDENTIALS_FILE"
 	snowCertificatesFile = "EKSA_AWS_CA_BUNDLES_FILE"
+	snowIPPoolIPStart    = "T_SNOW_IPPOOL_IPSTART"
```

Comment:
Do you think it's worth documenting what these values should be when configuring your snow e2e setup? I realize we don't actually have a lab to run against right now.

Reply:
updated readme: https://github.com/aws/eks-anywhere/pull/4979/files#diff-7d9a55ec9712c1ccc931efe126dbea316dab465e6ab265abd089736be5511f47R158